src/App.js

```diff
@@ -36,15 +41,24 @@ function App() {
         <h2>Enter a city below 👇</h2>
         <input
           type="text"
+          name="cityinbox"
           value={city}
           onChange={event => setCity(event.target.value)} />
+        <button onClick={() => setButtonClick(!buttonClick)}>Search</button>
+        <button onClick={() => setResults([])}>Clear All</button>
```

Comment:
To style the button, you can add a `className` to `<button>` tag and use that className to style in `App.css`